File: Application/RaceModel.cs
using RaceAnalysis.Domain.ValueTypes;
using System;
using System.Collections.Generic;
using System.Text;

namespace RaceAnalysis.Application
{
    public class RaceModel
    {
        public IEnumerable<PilotRaceStatsModel> PilotStats { get; set; }

        public LapRace  BestLapRace { get; set; }
    }
}

File: PresentationWeb/src/models/RaceModel.ts
import { PilotRaceStatsModel } from './PilotRaceStatsModel';
import { LapRaceModel } from './LapRaceModel';

export class RaceModel {
  public pilotStats: PilotRaceStatsModel[];
  public bestLapRace: LapRaceModel;
}
